CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### v1.4.1 Sept 06, 2022
+
+* Bump the minimum Reader SDK version support to 1.6.1 for iOS and 1.6.1 for Android
+* Update application UI
```

Comment:
What application? can we be more specific? I believe that's the quick start example UI?

Reply:
- [ ] update "Update application UI" to be more specific

Reply:
Done